sam/layers/README.md

```diff
@@ -0,0 +1,11 @@
+# Installing packages for Lambda Layers
+
+1. Create a folder per Lambda Layer (Note: There's a hard limit of 5 layers per function). Example, `requests` in order to create a layer that contains the `requests` package
+
+2. From each layer subdirectory, run:
+
+    ```Shell
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-content-indent] Don’t use mixed indentation for children, remove 1 space](https://app.codacy.com/manual/alanwill/slack-grim-reaper/pullRequest?prid=4805070)